docs/faq/questions/using-cypress-faq.mdx

```diff
@@ -469,8 +469,12 @@ it('fails on unhandled rejection', () => {
 
 ## <Icon name="angle-right" /> Can I override environment variables or create configuration for different environments?
 
-Yes, you can pass configuration to Cypress via environment variables, CLI
-arguments, JSON files and other means.
+Yes.
+
+- You can pass configuration to Cypress via environment variables, CLI
+  arguments, JSON files and other means.
+- You can override the config value for a **single** `cy.visit()` by setting the
```

Comment:
This update to the FAQ doesn't feel needed. There's lots of options that can be overridden besides setting `userAgent`, so I don't think we need to call it out explicitly here. Unless im missing something?

Reply:
I put this in the wrong section. Instead of the [Can I override environment variables or create configuration for different environments](https://docs.cypress.io/faq/questions/using-cypress-faq#Can-I-override-environment-variables-or-create-configuration-for-different-environments) section, it should have been under the [Can I override or change the default user agent the browser uses?](https://docs.cypress.io/faq/questions/using-cypress-faq#Can-I-override-or-change-the-default-user-agent-the-browser-uses) section